examples/stm32h7/src/bin/spi_bdma.rs

```diff
@@ -10,18 +10,24 @@ use embassy_executor::Executor;
 use embassy_stm32::mode::Async;
 use embassy_stm32::time::mhz;
 use embassy_stm32::{spi, Config};
+use grounded::uninit::GroundedArrayCell;
```

Comment:
Do you need to add `grounded` to the Cargo.toml as well?

Reply:
Ah! I didn't realize it was already there :D

https://github.com/embassy-rs/embassy/blob/main/examples/stm32h7/Cargo.toml#L37

Reply:
It has already been added:)
https://github.com/embassy-rs/embassy/commit/f91244bae6a4b277f8464c499e50c492a7671039